docs/superpowers/plans/2026-04-26-career-ops-web-ui.md

```
@@ -0,0 +1,2034 @@
+  },
+})
+```
+
+- [ ] **Step 5: Init Tailwind**
+
+```bash
+npx tailwindcss init -p --ts
+```
+
+Replace `tailwind.config.ts` content:
+```ts
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)